server_manager/electron/build.action.sh

```diff
@@ -72,4 +74,4 @@ cd "${STATIC_DIR}"
 
 # Icons.
 cd "${ROOT_DIR}"
-cp -r server_manager/electron/icons/ "${BUILD_DIR}/server_manager/electron/static/icons/"
+cp -r server_manager/electron/icons/ "${BUILD_DIR}/server_manager/${PLATFORM}/unpacked/icons/"
```

Comment:
I'm confused. Can you clarify the Electron build process?

I thought static was the output of building the web rederer code, and unpacked was the output of running Electron Builder. If so, you shouldn't be modifying unpacked, but giving the input to Electron Builder instead.

Reply:
Ah, ok, thanks for the clarification.